### PR TITLE
Feat: Salty server invite command

### PR DESCRIPTION
**Pull request by nebula231**
Adds command that provides discord invite link for Salty Simulations. Command aliases are .salty and .sal . It uses the salty simulations official discord server invite link 

image
zx#0671

### CODE REVIEW

**Comment by benw202** [comment]
Is this link a permanent invite? Do they have a vanity URL we can use instead?

**Comment by nebula231** [comment]
> Is this link a permanent invite? Do they have a vanity URL we can use instead?

It is a permanent invite link, not sure about the vanity. I just created it.

**Comment by Valastiri** [comment]
> > Is this link a permanent invite? Do they have a vanity URL we can use instead?
> 
> It is a permanent invite link, not sure about the vanity. I just created it.

By permanent/vanity we mean for example our is `https://discord.gg/flybywire`. We'd rather use this than user-generated ones.

File: src/commands/general/salty.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export const salty: CommandDefinition = {
+    name: ['salty', 'sal'],
+    description: 'Provides link to salty discord server',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        await msg.channel.send('https://discord.gg/S4PJDwk');
+        await msg.delete();
+    },
+};
```
Comment (NathanInnes) [inline src/commands/general/salty.ts:12]:
```suggestion
};

```

Add new line at the end

**Comment by NathanInnes** [comment]
> > > Is this link a permanent invite? Do they have a vanity URL we can use instead?
> > 
> > 
> > It is a permanent invite link, not sure about the vanity. I just created it.
> 
> By permanent/vanity we mean for example our is `https://discord.gg/flybywire`. We'd rather use this than user-generated ones.

They do not have a vanity URL

**Comment by nebula231** [comment]
> > > > Is this link a permanent invite? Do they have a vanity URL we can use instead?
> > > 
> > > 
> > > It is a permanent invite link, not sure about the vanity. I just created it.
> > 
> > 
> > By permanent/vanity we mean for example our is `https://discord.gg/flybywire`. We'd rather use this than user-generated ones.
> 
> They do not have a vanity URL



> > > > Is this link a permanent invite? Do they have a vanity URL we can use instead?
> > > 
> > > 
> > > It is a permanent invite link, not sure about the vanity. I just created it.
> > 
> > 
> > By permanent/vanity we mean for example our is `https://discord.gg/flybywire`. We'd rather use this than user-generated ones.
> 
> They do not have a vanity URL

I added line

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**